### PR TITLE
payload: LocKey fields, Map() test, and Browser payload

### PR DESCRIPTION
**Pull request by nathany**
* [x] LocKey
* [x] custom fields
* [x] Safari

### CODE REVIEW

File: payload/aps_test.go [hunk 1]
```diff
@@ -24,13 +24,19 @@ func TestBadgeAndSound(t *testing.T) {
...
 func TestContentAvailable(t *testing.T) {
-	p := payload.APS{
-		ContentAvailable: true,
-	}
+	p := payload.APS{ContentAvailable: true}
 	expected := []byte(`{"aps":{"content-available":1}}`)
 	testPayload(t, p, expected)
 }
 
+func TestCustomArray(t *testing.T) {
```
Comment (curtisallen) [inline payload/aps_test.go]: #picky this file looks like a great candidate for [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests)

**Comment by curtisallen** [comment]
Other then a small nit pick LGTM :+1:

**Comment by nathany** [comment]
please take a look (PTAL)

**Comment by curtisallen** [comment]
LGTM :+1:

**Pull request merged**